docs/cristin.ttl

```diff
@@ -16,35 +16,55 @@
 :Published
     a               rdfs:Class ;
     rdfs:label      "PUBLISHED" ;
```

Comment:
The data model doesn't have an english label for this field (not as a key, at least), so I guess remove this line?